tests/test_gbdesAstrometricFit.py

```diff
@@ -540,6 +570,285 @@ def test_run(self):
                     np.testing.assert_array_less(absDiffY, 1e-7)
 
 
+class TestGbdesGlobalAstrometricFit(TestGbdesAstrometricFit):
+    @classmethod
+    def setUpClass(cls):
+        # Set random seed
+        np.random.seed(1234)
+
+        # Fraction of simulated stars in the reference catalog and science
```

Comment:
What is the meaning/purpose of "simulated" stars here?

Reply:
I moved this block later so it should make more sense.